README.md

```diff
@@ -1,2 +1,5 @@
 # My-first-repository
 Repository containing all information needed
+Hello People,
+
+I am Suhas, I am from India. I am currently studying in Hamburg, Germany. I am an enthusianstic programmer. My hobbies are cooking, going for an adventurous trip, trying crazy stuffs, like to do something out of comfort zone.
```

Comment:
I have experience working at Dassault Systemes, India, with a programming knowledge in C++, JAVA. Could create some basic web pages too.